src/lein_monolith/config.clj

```diff
@@ -38,6 +38,11 @@
           next-files)))))
 
 
+(defn- add-raw
+  [monolith raw]
+  (assoc-in monolith [:monolith :raw] raw))
```

Comment:
Is it okay to be adding this to the `:monolith` map? Adding the raw project to the project map was simpler than changing the return type of the `find-monolith` function, but this does add some random metadata to the map…which makes me thing that we should just use `vary-meta`.